docs/content/documentation/systems/Kernels/framework/BodyForce.md

```diff
@@ -3,9 +3,26 @@
 ## Description
 
 `BodyForce` implements a force term in momentum transport or structural
-mechanics or a source term in species/mass transport. Its weak form looks like
-$$(\psi_i, -f)$$ where $f$ is the magnitude of the force or source. The force is
-constructed through a user supplied constant,
+mechanics or a source term in species/mass transport. The strong form, given a
+domain $\Omega$ is defined as
+
+\begin{equation}
+-f = 0 \in \Omega
```

Comment:
So I've seen this in some other documentation, and I'm not sure I agree with it. We're not trying to solve the equation f = 0 in Omega; e.g. this statement is actually saying our function is 0 in Omega. We're trying to solve "f + other\_terms = 0" in Omega.